storagenode/storagenodedb/testdata/sqlite.v19.sql

```diff
@@ -0,0 +1,201 @@
+    satellite_id  BLOB    NOT NULL,
+    action        INTEGER NOT NULL,
+    amount        BIGINT  NOT NULL,
+    created_at    TIMESTAMP NOT NULL
+);
+CREATE INDEX idx_bandwidth_usage_satellite ON bandwidth_usage(satellite_id);
+CREATE INDEX idx_bandwidth_usage_created   ON bandwidth_usage(created_at);
+
+-- table for storing all unsent orders
+CREATE TABLE unsent_order (
+    satellite_id  BLOB NOT NULL,
+    serial_number BLOB NOT NULL,
+
+    order_limit_serialized BLOB      NOT NULL,
+    order_serialized       BLOB      NOT NULL,
+    order_limit_expiration TIMESTAMP NOT NULL,
+
+    uplink_cert_id INTEGER NOT NULL,
+
+    FOREIGN KEY(uplink_cert_id) REFERENCES certificate(cert_id)
+);
+CREATE UNIQUE INDEX idx_orders ON unsent_order(satellite_id, serial_number);
+
+-- table for storing all sent orders
+CREATE TABLE order_archive_ (
+    satellite_id  BLOB NOT NULL,
+    serial_number BLOB NOT NULL,
+
+    order_limit_serialized BLOB NOT NULL,
+    order_serialized       BLOB NOT NULL,
+
+    uplink_cert_id INTEGER NOT NULL,
+
+    status      INTEGER   NOT NULL,
+    archived_at TIMESTAMP NOT NULL,
+
+    FOREIGN KEY(uplink_cert_id) REFERENCES certificate(cert_id)
+);
+
+CREATE TABLE bandwidth_usage_rollups (
+    interval_start	TIMESTAMP NOT NULL,
+    satellite_id  	BLOB    NOT NULL,
+    action        	INTEGER NOT NULL,
+    amount        	BIGINT  NOT NULL,
+    PRIMARY KEY ( interval_start, satellite_id, action )
+);
+
+-- table to hold expiration data (and only expirations. no other pieceinfo)
+CREATE TABLE piece_expirations (
+    satellite_id       BLOB      NOT NULL,
+    piece_id           BLOB      NOT NULL,
+    piece_expiration   TIMESTAMP NOT NULL, -- date when it can be deleted
+    deletion_failed_at TIMESTAMP,
+    PRIMARY KEY ( satellite_id, piece_id )
+);
+CREATE INDEX idx_piece_expirations_piece_expiration ON piece_expirations(piece_expiration);
+CREATE INDEX idx_piece_expirations_deletion_failed_at ON piece_expirations(deletion_failed_at);
+
+-- tables to store nodestats cache
+CREATE TABLE reputation (
+    satellite_id BLOB NOT NULL,
+    uptime_success_count INTEGER NOT NULL,
+    uptime_total_count INTEGER NOT NULL,
+    uptime_reputation_alpha REAL NOT NULL,
+    uptime_reputation_beta REAL NOT NULL,
+    uptime_reputation_score REAL NOT NULL,
+    audit_success_count INTEGER NOT NULL,
+    audit_total_count INTEGER NOT NULL,
+    audit_reputation_alpha REAL NOT NULL,
+    audit_reputation_beta REAL NOT NULL,
+    audit_reputation_score REAL NOT NULL,
+    updated_at TIMESTAMP NOT NULL,
+    PRIMARY KEY (satellite_id)
+);
+
+CREATE TABLE storage_usage (
+    satellite_id BLOB NOT NULL,
+    at_rest_total REAL NOT NUll,
+    timestamp TIMESTAMP NOT NULL,
+    PRIMARY KEY (satellite_id, timestamp)
+);
+
+CREATE TABLE piece_space_used (
+    total INTEGER NOT NULL,
+	satellite_id BLOB
+);
+CREATE UNIQUE INDEX idx_piece_space_used_satellite_id ON piece_space_used(satellite_id);
+
+DROP TABLE reputation;
```

Comment:
Is this needed here?

Reply:
yeah, this shouldn't be needed

Reply:
@ifraixedes  there are 2 `create table` statements for `reputation` table: one without `disqualified` flag, and the second one recreating table with this flag after table drop

Reply:
@egonelbre so your point is that I can delete `table drop` and second `table creation` from v.19 testdata file and just directly modify the first `table creation`?

Reply:
Yes, the testdata should setup the schema as it is and not do any migrations.